lunabot_control/scripts/manual_controller.py

```diff
@@ -190,9 +196,9 @@ def joy_callback(self, joy):
 
                 # Take priority for right trigger. If it is nearly zero, use the left trigger instead
                 if (right_trigger_axis_normalized <= 0.01):
-                    effort_msg.excavate = -1 * constrain(left_trigger_axis_normalized)
+                    effort_msg.excavate = -1 * constrain_RPM(left_trigger_axis_normalized, 1000)
```

Comment:
Self._max_speed should either be the same value for all motors, or potentially 3k for drive and 5k for excavation. Deposition doesn't really matter, I don't expect it to be going fast